app/build.gradle

```diff
@@ -51,6 +51,9 @@ android {
         proguardFile 'proguard-rules.pro'
```

Comment:
Not  this line, but lines 27-34, we had an issue with JavaVersion 1_8, so we had to switch to JavaVersion 11. I wonder if it's worth updating both for future proofing? Or should we look more into what the current standard is?